### DISCUSSION TITLE
Azure AD logins with NocoDB [May 13]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2094] [Discord Threads]
> 🙋‍♂️ **beata**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### SSO is planned as part of enterprise edition. We havent started work on that yet. We're working to make the current product robust. Which SQL database are you using -  (SQLServer?)  
#  
> 🙋‍♂️ **beata**  
#### Yes, We've used SqlServer. Looking forward to the enterprise edition. Would there be a way around it, perhaps using noco JWT variables and/or Noco Auth API in the meantime?We're trying this as a pilot now to get the business approval for the broader use of NocoDB.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/972074991246016522)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
